cloud/edgecontroller/pkg/devicecontroller/client/client.go

```diff
@@ -0,0 +1,3 @@
+package client
```

Comment:
I suggest remove the edgecontroller in the path of `cloud/edgecontroller/pkg/devicecontroller/client...`, xref: https://github.com/kubeedge/kubeedge/pull/390

Reply:
And rename controller to edgecontroller

Reply:
Once #390 is merged, I can rebase

Reply:
renaming controller to edgecontroller I will handle in a separate PR. Have to be cautious while doing it.